### PR TITLE
`visit-tag` - Work around GitHub tag picker icon bug

### LINKED ISSUE #7491: `visit-tag` adds useless link when switching branch picker tabs
### Description

When you change from "Branches" to "Tags" tab, the icon also changes thus matching our selector and triggers the addition of `visit-tag` link:

https://github.com/refined-github/refined-github/assets/44045911/d65d69a8-4b06-4bbf-89ee-0386f3fae4b0

We could probably just check if the branch picker is open and do nothing,

### How to replicate the issue + URL

https://github.com/refined-github/refined-github

### Extension version

24.6.21

### Browser(s) used

Chrome

### PR DESCRIPTION
**Pull request by kidonng**
Fix #7491

I think I still remember how to do RGH PRs 😂 

## Test URLs

https://github.com/refined-github/refined-github

## Screenshot

https://github.com/refined-github/refined-github/assets/44045911/5d308061-c452-4fe2-bde2-895fcfd1dd89

### CODE REVIEW

**Comment by fregante** [comment]
This is caused by their own bug, showing `main` as a tag: 

Untitled

File: source/features/visit-tag.tsx [hunk 1]
```diff
@@ -10,6 +11,11 @@ import {wrapAll} from '../helpers/dom-utils.js';
 import {buildRepoURL} from '../github-helpers/index.js';
 
 async function addLink(branchSelector: HTMLButtonElement): Promise<void> {
+	// If the branch picker is open, do nothing #7491
+	if (elementExists('#selectPanel')) {
```
Comment (fregante) [inline source/features/visit-tag.tsx:15]: Could work but it's not great. Do you see anyway to tell whether we are on a tag without relying on the picker?

https://github.com/refined-github/refined-github/tree/24.6.25

Comment (kidonng) [reply source/features/visit-tag.tsx:15]: The only thing of interest I can find (on current GitHub.com) is some SSR hydration data:

```js
// On `isRepoRoot`
data = document.querySelector('[partial-name="repos-overview"] [data-target="react-partial.embeddedData"]').text
payload = JSON.parse(data).props.initialPayload

// On `isRepoTree`
data = document.querySelector('react-app [data-target="react-app.embeddedData"]').text
payload = JSON.parse(data).payload

payload.refInfo.refType // `branch` or `tag`
```

I'm not sure how reliable this is though (whether the DOM selector or JSON selector).

**Comment by kidonng** [comment]
While we are at it, I also updated the icon class to fix alignment issue.

**Review by fregante — APPROVED** [review]
Fine as is 👌

**Pull request merged**